src/modules/mavlink/mavlink_main.h

```diff
@@ -181,7 +181,7 @@ class Mavlink : public ModuleParams
 		MAVLINK_MODE_CONFIG,
 		MAVLINK_MODE_IRIDIUM,
 		MAVLINK_MODE_MINIMAL,
-
+		MAVLINK_MODE_EXTVISION,
```

Comment:
Minor detail: can you leave the empty line before MAVLINK_MODE_COUNT?

Reply:
The empty line fails the style check from the CI.